lib/taskana-core/src/main/java/pro/taskana/impl/TaskanaEngineImpl.java

```diff
@@ -232,6 +234,9 @@ protected SqlSessionManager createSqlSessionManager() {
           e.getCause());
     }
 
+    // register type handlers
+    configuration.getTypeHandlerRegistry().register(MapTypeHandler.class);
+    configuration.getTypeHandlerRegistry().register(Instant.class, InstantTypeHandler.class);
```

Comment:
Just out of curiosity: 
Should we instantiate the Handlers, so that mybatis doesnt do it via reflections?

Reply:
Does that even matter?

Reply:
I even don't know an interface that we could use to pass instantiated handlers to mybatis...

Reply:
https://mybatis.org/mybatis-3/apidocs/reference/org/apache/ibatis/type/TypeHandlerRegistry.html

would be
register(TypeHandler<T> typeHandler) for the MapTypeHandler
and 
register(Class<T> javaType, TypeHandler<? extends T> typeHandler) for the InstantTypeHandler

Reply:
looking at the sourcecode of mybatis the register(Class<T> ..) methods do some reflection magic and then call the regular register(TypeHandler<?> ..) methods. Do we want that reflection magic to happen?

Reply:
I'll see whether this works also and if so, register the instantiated typehandlers ...

Reply:
Works fine. I switched to the registration with instantiated handlers ...